src/app/(proper_react)/redesign/(authenticated)/user/(dashboard)/settings/AlertAddressForm.tsx

```diff
@@ -0,0 +1,114 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import {
+  AriaRadioProps,
+  useFocusRing,
+  useRadio,
+  useRadioGroup,
+} from "react-aria";
+import { RadioGroupState, useRadioGroupState } from "react-stately";
+import styles from "./AlertAddressForm.module.scss";
+import { useL10n } from "../../../../../../hooks/l10n";
+import { createContext, useContext, useRef } from "react";
+import type { EmailUpdateCommOptionRequest } from "../../../../../../api/v1/user/update-comm-option/route";
+import { VisuallyHidden } from "../../../../../../components/server/VisuallyHidden";
+
+export type AlertAddress = "primary" | "affected";
+
+export type Props = {
+  defaultSelected: AlertAddress;
+};
+
+const AlertAddressContext = createContext<RadioGroupState | null>(null);
+
+export const AlertAddressForm = (props: Props) => {
+  const l10n = useL10n();
+  const state = useRadioGroupState({
```

Comment:
It would make sense, but I think we probably want a completely new design here that don't involve form elements that do things without submitting a form. Since I'm mostly aiming for feature parity here to unblock the release, I think it's best to iterate on the design in a later stage.

Reply:
Getting up to feature parity sounds good to me.